app/Exceptions/Handler.php

```diff
@@ -34,34 +35,36 @@ public function report(Exception $e)
     /**
      * Render an exception into an HTTP response.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Exception  $e
+     * @param  \Illuminate\Http\Request $request
+     * @param  \Exception $e
      * @return \Illuminate\Http\Response
      */
     public function render($request, Exception $e)
     {
-        if (env('APP_DEBUG') != true) {
```

Comment:
Its changing the behaviour of the application based on an environment variable.
If you cannot be confident of what the application does while its on dev / prod  or with debug mode, then it reduces your confidence while doing changes or trying to replicate bugs locally. Instead I added logging for the unhappy paths.

If you really want to preserve this functionality let me know and I will revert that change.

😁